packages/contentstack-config/messages/index.json

```diff
@@ -12,5 +12,7 @@
   "CLI_CONFIG_SELECT_REGION": "Please select a region",
   "CLI_CONFIG_INQUIRE_API_KEY": "Enter stack api key",
   "CLI_CONFIG_INQUIRE_BASE_BRANCH": "Enter base branch",
-  "CLI_CONFIG_BRANCH_LIST_NO_BRANCHES": "No branches are added. Use config:set:base-branch command to add base branch."
+  "CLI_CONFIG_BRANCH_LIST_NO_BRANCHES": "No branches are added. Use config:set:base-branch command to add base branch.",
+  "CLI_CONFIG_INQUIRE_EARLY_ACCESS_HEADER_VALUE": "Enter early access header value",
+  "CLI_CONFIG_INQUIRE_EARLY_ACCESS_HEADER_ALIAS": "Enter early access header alias",
```

Comment:
@shafeeqd959 , Could you please check? It appears to be displaying 'CLI_CONFIG_INQUIRE_EARLY_ACCESS_HEADER_ALIAS' instead of "Enter early access header alias," which I think is due to the trail comma.

Reply:
done, thanks